coralproject/talk
# [CORL-501] Moderation Notifications

## What does this PR do?

Adds moderation comment notification support.

## How do I test this PR?

1. Using a admin user, set the whole site to premod.
2. Using a comment user, enable the notification type under your profile settings.
3. Write a comment.
4. Using a admin user, approve the comment.
5. Check your email!
6. Using a comment user, write a comment.
7. Using a admin user, rejected the comment.
8. Check your email!
